data/en/construction_0002.json

```diff
@@ -30,7 +31,7 @@
                             "id": "preview",
                             "title": "Information you need",
                             "content": [{
```

Comment:
rather than hacking together a single line for the url, you can use 2 content json objects like this
"content": [{
                                "description": "<a rel='noopener noreferrer' target='_blank' href='https://www.ons.gov.uk/surveys/informationforbusinesses/businesssurveys/monthlybusinesssurveyforconstructionandalliedtrades'>View the survey information and questions before you start the survey</a> (external link)"
                            },{
                                "description": "You will be asked to provide information on the value of work undertaken by your business and the number of employees. Value of work refers to estimates of chargeable work carried out during the month; meaning work started, in progress, or completed."
                            }]

Reply:
git hub has converted the url. use the <a> tags as currently is there.